runtime/doc/builtin.txt

```diff
@@ -4315,19 +4319,17 @@ getregion({pos1}, {pos2} [, {opts}])			*getregion()*
 		- If the selection starts or ends in the middle of a multibyte
 		  character, it is not included but its selected part is
 		  substituted with spaces.
-		- If {pos1} or {pos2} equals "v" (see |line()|) and it is not in
-		  |visual-mode|, an empty list is returned.
-		- If {pos1}, {pos2} or {type} is an invalid string, an empty
-		  list is returned.
-		- If {pos1} or {pos2} is a mark in different buffer, an empty
-		  list is returned.
+		- If {type} is an invalid string, an empty list is returned.
```

Comment:
`{type}` is in the dict now, and here it is referenced as a standalone parameter of a function.